### PR TITLE
Clean up and test model

### LINKED ISSUE #56: Cleanup & Test all models
- [x] Public methods of all models e.g. Log, Rep, Name etc

### PR DESCRIPTION
**Pull request by Jillzyt**
- Passed CheckstyleMain and CheckStyleTest.
- Deleted address and phone in Log folder
- Did not touch NameContainsKeywordsPredicate as it will overlap with addressbook
- Did not touch NameContainsKeywordsPredicateTestCase as it will overlap with addressbook

### CODE REVIEW

File: src/main/java/seedu/fma/model/log/UniqueLogList.java [hunk 1]
```diff
@@ -79,7 +79,7 @@ public void remove(Log toRemove) {
         }
     }
 
-    public void setPersons(UniquePersonList replacement) {
+    public void setPersons(UniqueLogList replacement) {
```
Comment (therizhao) [inline src/main/java/seedu/fma/model/log/UniqueLogList.java]: should be setLogs?

need to fix the naming of the overloaded method below too

File: src/test/java/seedu/fma/model/exercise/ExerciseTest.java [hunk 2]
```diff
@@ -31,4 +48,17 @@ public void equals() {
         Exercise editedSitUp = new ExerciseBuilder(SIT_UP).withName(VALID_EXERCISE_CRUNCHES).build();
         assertFalse(SIT_UP.equals(editedSitUp));
     }
+
+    @Test
+    public void finds() {
```
Comment (therizhao) [inline src/test/java/seedu/fma/model/exercise/ExerciseTest.java]: should be find w/o s

can split this into 2 tests

find_state_expected

File: src/test/java/seedu/fma/model/log/CommentTest.java [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+public class CommentTest {
+
+    @Test
+    public void constructor_null_throwsNullPointerException() {
+        assertThrows(NullPointerException.class, () -> new Comment(null));
+    }
+
+
+    @Test
+    public void isValidComment() {
```
Comment (therizhao) [inline src/test/java/seedu/fma/model/log/CommentTest.java]: Can split into multiple tests 
with format 
method_state_expected

**Review by youaremysky99 — DISMISSED** [review]
LGTM

**Review by therizhao — CHANGES_REQUESTED** [review]
minor fixes

File: src/test/java/seedu/fma/model/log/RepTest.java [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+    }
+
+    @Test
+    public void constructor_invalidRep_throwsIllegalArgumentException() {
+        String invalidRep = "";
+        assertThrows(IllegalArgumentException.class, () -> new Rep(invalidRep));
+    }
+
+    @Test
+    public void isValidRep() {
```
Comment (therizhao) [inline src/test/java/seedu/fma/model/log/RepTest.java]: Can split into multiple tests
with format
method_state_expected

**Review by therizhao — CHANGES_REQUESTED** [review]

File: src/main/java/seedu/fma/model/log/UniqueLogList.java (hunk 1 above)
Comment (Jillzyt) [reply src/main/java/seedu/fma/model/log/UniqueLogList.java]: Updated the method name.

File: src/test/java/seedu/fma/model/exercise/ExerciseTest.java (hunk 2 above)
Comment (Jillzyt) [reply src/test/java/seedu/fma/model/exercise/ExerciseTest.java]: Updated

File: src/test/java/seedu/fma/model/log/CommentTest.java (hunk 3 above)
Comment (Jillzyt) [reply src/test/java/seedu/fma/model/log/CommentTest.java]: Updated

File: src/test/java/seedu/fma/model/log/RepTest.java (hunk 4 above)
Comment (Jillzyt) [reply src/test/java/seedu/fma/model/log/RepTest.java]: Updated

**Comment by Jillzyt** [comment]
Passed checkstyleMain, checkstyleTest.

**Review by therizhao — APPROVED** [review]
lgtm

**Pull request merged**